compiler/one-cmds/tests/onnx_legalize_run_compare.py

```diff
@@ -0,0 +1,71 @@
+import onnxruntime as rt
+import onnx
+import sys
+import os
+import numpy as np
+
+def _generate_inputs(model):
+    inputs = {}
+    for input in model.graph.input:
+        # check if elem type is float32
+        assert(input.type.tensor_type.elem_type == 1)
+        input_shape = []
+        for dim in input.type.tensor_type.shape.dim:
+            input_shape += [dim.dim_value]
+        inputs[input.name] = np.random.random(input_shape).astype(np.float32)
+    return inputs
+
+def _run_model(model, inputs):
+    output_names = list(map(lambda output: output.name, model.graph.output))
+    session = rt.InferenceSession(model.SerializeToString())
+    outputs = session.run(output_names, inputs)
+    return outputs
+
+def _compare_resuts(ref_outputs, test_outputs, tolerance):
+    num_outputs = len(ref_outputs)
+    assert(len(test_outputs) == num_outputs)
+    for i in range(num_outputs):
+        if ref_outputs[i].shape != test_outputs[i].shape:
+            print("output {} shape mismatch".format(i))
+            return False
+        peak_error = np.abs(ref_outputs[i] - test_outputs[i]).max()/np.abs(ref_outputs[i]).max()
+        if peak_error > tolerance:
+            print("output {} peak error to value ratio {} is too big".format(i, peak_error))
+            return False
+    return True
+
+if __name__ == '__main__':
+    # this manipulation is needed to add searcg path where onnx_legalizer is installed
+    sys.path += os.environ['PATH'].split(':')
+    import onnx_legalizer
+
+
```

Comment:
(does format checker require two lines after import? if not can we remove one?)

Reply:
Oof, I just noticed that #8221 disabled checks for compiler/one-cmds directory: [these lines](https://github.com/Samsung/ONE/pull/8221/files#diff-f26ca7df49b7cc7c3f32f3769aa92113e722738d77a5f2458a5850dde86b352dR181-R182)

Reply:
created issue for this: #8448